README.md

```diff
@@ -93,7 +93,7 @@ func (s *Session) Rcpt(to string) error {
 	return nil
 }
 
-func (s *Session) Data(r io.Reader) error {
+func (s *Session) Data(r io.Reader, sc smtp.LMTPStatusCollector) error {
```

Comment:
Could rename to a neutral name like `smtp.DataContext` for future extensibility.

Reply:
ok fine!

Reply:
DONE with map (see new pull req)